class-wc-siftscience-dependencies.php

```diff
@@ -98,8 +98,17 @@ private function build( $class ) {
 			}
 
 			$args = array();
-			foreach ( $c->getParameters() as $p ) {
-				$args[] = $this->get( $p->getType()->getName() );
+
+			$php_version = explode( '.', phpversion() );
```

Comment:
Technocally $php_version is a string array. Should we explicitly cast to int inside the if-statement?

Reply:
Idk

Reply:
Why

Reply:
1 < '1' --> Is this true or false?

12 < '9' --> is this going to be counted as:

-  '12' < '9' (cast as strings) --> true
- 12 < 9 (cast as int) --> false

I don't remember what PHP does by default, which is why I prefer to explicitly cast the type.